### PR TITLE
Upgrade expo to v39

### PR DESCRIPTION
**Pull request by cornedor**
sentry-expo has no update yet.

Also upgraded react-navigation and react-native-keyboard-aware-scroll-view to fix issues with add emission screen

Fixes https://github.com/NMF-earth/nmf-app/issues/164

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -25,24 +25,24 @@
     ]
   },
   "dependencies": {
-    "@expo/vector-icons": "10.0.0",
-    "@react-native-community/datetimepicker": "2.4.0",
+    "@expo/vector-icons": "^10.0.0",
```
Comment (PierreBresson) [inline package.json:28]: No ^ so we all have the same version

File: package.json [hunk 2]
```diff
@@ -25,24 +25,24 @@
...
-    "expo": "38.0.0",
-    "expo-asset": "8.1.7",
-    "expo-constants": "9.1.1",
-    "expo-font": "8.2.1",
-    "expo-localization": "8.2.1",
-    "expo-splash-screen": "0.5.0",
-    "expo-status-bar": "1.0.2",
-    "expo-web-browser": "8.3.1",
+    "expo": "^39.0.0",
+    "expo-asset": "~8.2.0",
```
Comment (PierreBresson) [inline package.json:39]: No ~ so we all have the same version

**Review by PierreBresson — CHANGES_REQUESTED** [review]
Almost good 🙂

File: package.json (hunk 1 above)
Comment (cornedor) [reply package.json:28]: See this comment: https://github.com/NMF-earth/nmf-app/pull/181#discussion_r500784723

Comment (cornedor) [reply package.json]: I've locked the versions

**Comment by PierreBresson** [comment]
@cornedor Dank!

**Pull request merged**